packages/salesforcedx-vscode-core/test/vscode-integration/context/workspaceOrgType.test.ts

```diff
@@ -76,9 +60,6 @@ describe('getWorkspaceOrgType', () => {
     const orgType = await getWorkspaceOrgType(defaultUsername);
 
     expect(orgType).to.equal(OrgType.NonSourceTracked);
-    expect(authInfoCreateStub.getCall(0).args[0]).to.eql({
-      username: undefined
```

Comment:
Should we now expect that the username here is no longer undefined? I'm not certain about removing this assertion altogether.